evap/static/scss/components/_grid.scss

```diff
@@ -155,7 +155,7 @@
 }
 
 th.col-order {
-    padding-left: 1.4em;
+    padding-left: 1.4em !important;
```

Comment:
@richardebeling This is now the same as in the rule for just `.col-order`, so we should be able to just remove this rule altogether, right? I am not sure why exactly we had explicitly different values for the `th` and non-`th` rules, maybe @janno42 can remember?

Reply:
we probably needed `th.col-order` to be more specific than something else to avoid using `!important`. removing this line would require us to put `!important` behind the rule in `.col-order`.
i would prefer a solution without `!important` by adding the other classes of the object to the selector for higher specificity (https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)